docs/src/further_topics/controlling_merge.rst

```diff
@@ -0,0 +1,198 @@
+Each of these files covers a month's time step, however, the ``surface_altitude`` coordinate changes every year. The
+files span 3 years so there are 3 different "surface_altitude" coordinates.
+
+    >>> filename = iris.sample_data_path('time_varying_hybrid_height', '*.pp')
+    >>> cubes = iris.load(filename, constraints="x_wind")
+    >>> print(cubes)
+    0: x_wind / (m s-1)                    (time: 2; model_level_number: 5; latitude: 144; longitude: 192)
+    1: x_wind / (m s-1)                    (time: 12; model_level_number: 5; latitude: 144; longitude: 192)
+    2: x_wind / (m s-1)                    (model_level_number: 5; latitude: 144; longitude: 192)
+
+When :func:`iris.load` attempts to merge these cubes, it creates a cube for every unique ``surface_altitude`` coordinate.
+Note that since there is only one time point associated with the last cube, the "time" coordinate has not been promoted
+to a dimension. The ``surface_altitude`` in each of the above cubes is 2D, however, since some of these coordinates
+already have a time dimension, it is not possible to use :func:`~iris.util.new_axis` as above to promote
+``surface_altitude`` as we have done above. In order to fully control the merge process we instead use
+:func:`iris.load_raw`:
+
+    >>> raw_cubes = iris.load_raw(filename, constraints="x_wind")
+    >>> print(raw_cubes)
+    0: x_wind / (m s-1)                    (latitude: 144; longitude: 192)
+    1: x_wind / (m s-1)                    (latitude: 144; longitude: 192)
+    ...
+    73: x_wind / (m s-1)                    (latitude: 144; longitude: 192)
+    74: x_wind / (m s-1)                    (latitude: 144; longitude: 192)
+
+The raw cubes also separate cubes along the ``model_level_number`` dimension. In this instance, we will need to
+merge/concatenate along two different dimensions. Specifically, we can merge along the ``model_level_number`` dimension
+since ``surface_altitude`` does  not vary along this dimension, and we can concatenate along the ``time`` dimension as
+before. We expand the ``time`` dimension first, as before:
+
+    >>> processed_raw_cubes = CubeList(
+    ... [new_axis(cube, scalar_coord="time", expand_extras=["surface_altitude", "forecast_period"]) for cube in raw_cubes]
+    ... )
+    >>> print(processed_raw_cubes)
+    0: x_wind / (m s-1)                    (time: 1; latitude: 144; longitude: 192)
+    1: x_wind / (m s-1)                    (time: 1; latitude: 144; longitude: 192)
+    ...
+    73: x_wind / (m s-1)                    (time: 1; latitude: 144; longitude: 192)
+    74: x_wind / (m s-1)                    (time: 1; latitude: 144; longitude: 192)
+
+Then we merge along the ``model_level_number`` dimension. Note that merging these cubes does not affect the ``time``
+dimension since merging only applies along scalar coordinates, not dimension coordinates of length 1.
```

Comment:
```suggestion
Then we merge, which will create a new ``model_level_number`` dimension, since there is one cube for each level, at each time point.  Note, however, that merging these cubes does _not_ affect the ``time``
dimension, since merging only applies to scalar coordinates, not dimension coordinates of length 1.
```

I'm uncomfortable with "merge along the `model_level_number` dimension" since, of course, merge chooses the merge dimensions automatically +  we have no control over it.
I also generally don't like "merge along" because we don't have a dimension giving a direction to be "along", whereas (for me) "concatenate along" **_is_** clear + meaningful. 

Not entirely sure about my alternative proposals, though.  Maybe you can think of something better.